pom.xml

```diff
@@ -235,7 +235,8 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-surefire-plugin</artifactId>
-					<version>2.21.0</version>
+					<!-- version>2.21.0</versioni-->
+			                <version>2.21.0</version>
```

Comment:
Please refrain from pom.xml changes. This issue is about the Couchbase database plugin. We can sort out any checkstyle warnings later on.

Reply:
Yes, sorry, I got in a real Maven mess as a work proxy was corrupting downloads and causing confusing errors which I tried to resolve by commenting stuff out before I figured it out. Have reverted.